src/components/Sum/Sum.jsx

```diff
@@ -1 +1,9 @@
-// export const Sum = () => ();
+// import { sum } from "cypress/types/lodash";
+
+export const Sum = ({ a = 0, b = 0 }) => (
+  <>
+    <p>
+      {`Sum of ${a} and ${b} is ${a + b}`}
+    </p>
+  </>
```

Comment:
no need fragment here

```suggestion
    <p>
      {`Sum of ${a} and ${b} is ${a + b}`}
    </p>
```